src/lib/remark-plugins/remark-plugin-adjust-link-urls/helpers.ts

```diff
@@ -84,6 +75,44 @@ const handleDotSlashFolderRelativeUrl = ({
 	return newUrl
 }
 
+/**
+ * Handles folder-relative URLs that do not start with any dots syntax or
+ * punctuation.
```

Comment:
```suggestion
 * Handles folder-relative URLs that do not start with any dots syntax or
 * punctuation. For example: api-docs/some-path 
```
An example could be nice!

Reply:
Updated here: https://github.com/hashicorp/dev-portal/pull/1544/commits/73d5a96615d7192c2e24b7eff5de35f93eda2a06